k8s/aat/common/ccpay/ccpay-bulkscanning-app.yaml

```
@@ -0,0 +1,37 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccpay-bulkscanning-api
+  namespace: fees-pay
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ccpay-bulkscanning-api
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ccpay-bulkscanning-api
+    version: 0.0.4
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "280b07a1-f1a1-4369-8e21-0bb2c2995f1e"
+      image: hmctspublic.azurecr.io/hmcts/ccpay-bulkscanning-api:prod-*
+      ingressHost: ccpay-bulkscanning-api.service.core-compute-aat.internal
+      environment:
+       POSTGRES_HOST: ccpay-bulkscanning-api-postgres-db-aat.postgres.database.azure.com
+       POSTGRES_NAME: bspayment
+       POSTGRES_USER: bspayment@ccpay-bulkscanning-api-postgres-db-aat
+      aadIdentityName: ccpay
+      keyVaults:
+        "ccpay":
+          resourceGroup: ccpay
+          secrets:
+            - bulkscanning-api-POSTGRES-PASS
+            - AppInsightsInstrumentationKey
+    global:
+      environment: aat
+      subscriptionId: "96c274ce-846d-4e48-89a7-d528432298a7"
```

This should be subscription id of the vault you want to access.
```suggestion
      subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
```